src/core_plugins/input_control_vis/public/register_vis.js

```diff
@@ -73,6 +72,12 @@ function InputControlVisProvider(Private) {
     requestHandler: 'none',
     responseHandler: 'none',
   });
+
+  inputControlsVisType.inspectorAdapters = {
```

Comment:
This should rather be in the description above. Is there a specific reason that was pulled out? If there is, I think it's rather an issue due to some refactoring, that we should fix.